runner/src/mill/runner/MillCliConfig.scala

```diff
@@ -117,7 +117,15 @@ class MillCliConfig private (
            code changes, and instead fall back to the previous coarse-grained implementation
            relying on the script `import $file` graph"""
     )
-    val disableCallgraphInvalidation: Flag
+    val disableCallgraphInvalidation: Flag,
+    @arg(
+      name = "meta-level",
+      doc =
+        """Experimental: Select a meta-build level to run the given targets.
+           Level 0 is the normal project, level 1 the first meta-build, and so on.
+           The last level is the built-in synthetic meta-build which Mill uses to bootstrap the project."""
+    )
+    val metaLevel: Option[Int]
```

Comment:
Is there a reason why we don't use default values here? `val metaLevel: Int = 0` which should be supported by mainargs?

Reply:
Defaults are provided in the `apply` method. 

```scala
frame: Option[Int] = None
```

Using `Option[Int]` in contrast to `Int` is more explicit whether the user provided a value or not.